test/signs_ui/signs/expiration_test.exs

```diff
@@ -0,0 +1,74 @@
+defmodule SignsUI.Signs.ExpirationTest do
+  use ExUnit.Case
+  alias SignsUI.Signs.Sign
+
+  describe "expire_signs/2" do
+    test "expire_signs/2" do
```

Comment:
I like it. 👍

Reply:
Heh, ooops. I renamed the `test` to something a bit more descriptive

Reply:
Haha, I meant the refactored test, not just a sarcastic snark on the test name!